### PR TITLE
SASS-85 added beta banner

### PR DESCRIPTION
**Pull request by juamalBlackman**

### CODE REVIEW

File: app/views/headerFooterTemplate/Head.scala.html [hunk 1]
```diff
@@ -21,3 +21,4 @@
 <!--[if lte IE 8]><script src='@controllers.routes.Assets.versioned("javascripts/html5shiv.min.js")'></script><![endif]-->
 <!--[if lte IE 8]><link href='@controllers.routes.Assets.versioned("stylesheets/application-ie-8.css")' rel="stylesheet" type="text/css" /><![endif]-->
 <!--[if gt IE 8]><!--><link href='@controllers.routes.Assets.versioned("stylesheets/application.css")' media="screen" rel="stylesheet" type="text/css" /><!--<![endif]-->
+<link rel="stylesheet" href=@routes.Assets.versioned("stylesheets/banner-panel.css")>
```
Comment (robert-webb-hmrc) [inline app/views/headerFooterTemplate/Head.scala.html:24]: I would double check this, as it currently stand the pages do not load this css file in.

File: app/views/templates/helpers/BetaBar.scala.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+@()
+
+<div class="govuk-phase-banner">
+    <p class="govuk-phase-banner__content">
+        <strong class="govuk-tag govuk-phase-banner__content__tag">
+      beta
+        </strong>
+        <span class="govuk-phase-banner__text">
+      This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.
```
Comment (robert-webb-hmrc) [inline app/views/templates/helpers/BetaBar.scala.html:27]: Would it make sense for these to be in the messages file?

File: app/views/dividends/OtherDividendsAmountView.scala.html [hunk 3]
```diff
@@ -23,17 +23,19 @@
 @import views.html.templates.helpers.ErrorSummaryDisplay
 @import views.html.templates.helpers.BackLink
 @import views.html.templates.helpers.ContinueButton
+@import views.html.templates.helpers.BetaBar
 
-@this(layout: Layout, backLink: BackLink, formWithCSRF: FormWithCSRF, errorSummaryDisplay: ErrorSummaryDisplay )
+
+@this(layout: Layout, backLink: BackLink, formWithCSRF: FormWithCSRF, errorSummaryDisplay: ErrorSummaryDisplay, betaBar: BetaBar )
```
Comment (robert-webb-hmrc) [inline app/views/dividends/OtherDividendsAmountView.scala.html:29]: do you need to pull in the betaBar here? This is the only view in which you are doing so, I assume that @layout covers this

**Review by robert-webb-hmrc — CHANGES_REQUESTED** [review]

File: app/views/headerFooterTemplate/Head.scala.html (hunk 1 above)
Comment (robert-webb-hmrc) [reply app/views/headerFooterTemplate/Head.scala.html:24]: Screenshot 2020-10-27 at 11 40 21

Comment (juamalBlackman) [reply app/views/headerFooterTemplate/Head.scala.html]: removed

File: app/views/templates/helpers/BetaBar.scala.html (hunk 2 above)
Comment (juamalBlackman) [reply app/views/templates/helpers/BetaBar.scala.html]: added to messages

File: app/views/dividends/OtherDividendsAmountView.scala.html (hunk 3 above)
Comment (juamalBlackman) [reply app/views/dividends/OtherDividendsAmountView.scala.html]: removed all backlinks from view files along with betabars

**Review by robert-webb-hmrc — APPROVED** [review]

**Pull request merged**